boot/zephyr/Kconfig

```diff
@@ -364,6 +370,50 @@ config BOOT_WAIT_FOR_USB_DFU
 	  for USB DFU to be invoked. Please note DFU always updates the
 	  slot1 image.
 
+config BOOT_USB_DFU_WAIT_DELAY_MS
+	int
+	depends on BOOT_USB_DFU_WAIT
+	default 12000
+	help
+	  The duration to wait for USB DFU to be invoked.
```

Comment:
I know that the config identifier states the unit is MS, but maybe it is worth mentioning that these are milliseconds in the help.